cloudtrust-common/src/test/java/io/cloudtrust/utils/CryptoUtilTest.java

```diff
@@ -37,7 +37,7 @@ public void init() {
     public void testSecretEncryptionDecryption() {
         byte[] plainText = new byte[32]; //256 bits
         new SecureRandom().nextBytes(plainText);
-        assertThat(CryptoUtil.decryptFromDatabaseStorage(CryptoUtil.encryptForDatabaseStorage(plainText)), equalTo(plainText));
+        org.hamcrest.MatcherAssert.assertThat(CryptoUtil.decryptFromDatabaseStorage(CryptoUtil.encryptForDatabaseStorage(plainText)), equalTo(plainText));
```

Comment:
Why have the full class package, as it is anyway imported?

Reply:
Absolutely no idea!